test/e2e/flask/multichain-api/testHelpers.ts

```diff
@@ -65,7 +72,16 @@ export async function createSessionScopes(
   const editButtons = await driver.findElements('[data-testid="edit"]');
   await editButtons[1].click();
   await driver.delay(largeDelayMs);
+}
 
+/**
+ * Confirms wallet operation and switches focus to multichain test dapp
+ *
+ * @param driver - E2E test driver {@link Driver}, wrapping the Selenium WebDriver.
+ */
+export async function confirmAndSwitchFocusToDapp(
+  driver: Driver,
+): Promise<void> {
```

Comment:
Decided to separate the creation of session scopes from the confirmation and window focus switch with `confirmAndSwitchFocusToDapp`, since we needed to pause for a bit to assert some conditions in one of the added test cases.

Reply:
Not sure if this needs its own helper. Or at least I think this function doesn't do enough to warrant not just being inlined in the test

Reply:
it was a bit more complex when first created, but now it makes sense to just inline this 👍🏾 

resolved in [422b254](https://github.com/MetaMask/metamask-extension/pull/29356/commits/422b254773666c06c7905f2fd397888f52da9453)